hack/list-images.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly SCRIPT_DIR
+REPO_ROOT="$(realpath "$(dirname "${SCRIPT_DIR}")")"
+readonly REPO_ROOT
+pushd "${REPO_ROOT}" &>/dev/null
+
+trap_add() {
+  local -r sig="${2:?Signal required}"
+  local -r hdls="$(trap -p "${sig}" | cut --fields=2 --delimiter=\')"
+  # shellcheck disable=SC2064 # Quotes are required here to properly expand when adding the new trap.
+  trap "${hdls}${hdls:+;}${1:?Handler required}" "${sig}"
+}
+
+while IFS= read -r repofile; do
+  envsubst -no-unset -no-digit -i "${repofile}" | \
+    gojq --yaml-input --raw-output 'select(.spec.url != null) | (.metadata.name | gsub("\\."; "-"))+" "+.spec.url' | \
+    xargs --max-lines=1 --no-run-if-empty -- helm repo add --force-update >&2
+done < <(grep --recursive --max-count=1 --files-with-matches '^kind: HelmRepository')
+
+helm repo update >&2
+
+# Dummy variables
+declare -rx releaseNamespace=unused \
```

Comment:
Will this script fail if a new var is introduced and not added to this list?

Reply:
Yes, the use of `envsubst` `--no-unset` flag as in https://github.com/mesosphere/kommander-applications/pull/1382/files#diff-92a71aa3a9751144ef83a5eb68635485897b33738aa7a3759833ddf27848d9beR19 will mean that if an unknown var is added it will fail. This is what we want to ensure all vars are considered.